.github/ISSUE_TEMPLATE/spike.yml

```
@@ -0,0 +1,37 @@
+name: "Spike Task"
+description: "Research task for technical exploration or feasibility analysis."
+labels: ["spike"]
+body:
+    -   type: textarea
```

YAMLlint (1.35.1)

[error] too many spaces after hyphen

(hyphens)